packages/frontend/src/App.styles.tsx

```diff
@@ -1,6 +1,16 @@
+import { Colors } from '@blueprintjs/core';
 import { createGlobalStyle } from 'styled-components';
 
 const AppStyle = createGlobalStyle`
+    .bp4-multi-select-tag-input-input::placeholder {
```

Comment:
this will  affect all multi-selects and dialogs, is this what we want ?

Reply:
good question.. I guess we should change all (including other kinds of inputs from blueprint). 
@PriPatel wdyt? should we change the colour for all input placeholders or just this one. If yes - then im gonna create a separate issue for it.